demo/src/Decoder/index.tsx

```diff
@@ -20,6 +20,15 @@ const DecoderPanel = ({ width }: { width: string | number }) => {
       <DecoderCard setDecSecret={getDecSecret} />
 
       <DecodeSecretCard decSecret={decSecret} />
+
+      {/* <Image
```

Comment:
delete useless comments.

if you would need this during recent dev, add a 'TODO:' tag for later reminding for deteting.